core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-box/template-builder-box.component.html

```diff
@@ -0,0 +1,59 @@
+<ng-template [ngIf]="size !== 'small'">
+    <p-card
+        [ngClass]="{
+            'template-builder-box__medium': size === 'medium',
```

Comment:
In BEM standard, the variations are represented using `--` so in this case should be `template-builder-box--medium`